ansible/configs/sap-integration/default_vars.yml

```diff
@@ -0,0 +1,384 @@
+# This should almost always be set to true for OpenShift installations
+# If it is set to false, the {{ osp_project_name }} must already exist and
+# should be able to run whatever you are deploying
+osp_project_create: true
+
+
+# The name of the OpenShift cluster that will be deployed.
+# This is primarily used if you want to automate the OpenShift deployment.
+cluster_name: "cluster-{{ guid }}"
+
+# Used to add metadata (tags) to OpenStack objects created
+project_tag: "{{ env_type }}-{{ guid }}"
+
+# Why is this config being deployed? Override for specific environments
+# Some valid: development, ilt, production, event
+purpose: development
+
+# Enable this if you want to use IPA for user authentication.
+# Mutually exclusive with {{ install_student_user }}
+install_ipa_client: false
+
+# Enable this if you want to create a user on the bastion
+# Mutually exclusive with {{ install_ipa_client }}
+install_student_user: false
+
+# TODO: What does this really do besides run the role?
+set_env_authorized_key: true
+env_authorized_key: "{{guid}}key"
+key_name: "default_key_name"
+
+# Run the bastion-lite role
+install_bastion: true
+
+# This config was written with the expectation of using python3
+# Several of the roles that it depends on only work with python2,
+# so they had to be modified to handle python3 as well. Use this
+# var so that it is passed along and influences how those roles run.
+all_use_python3: true
+
+# FTL is used for grading and solving. It will pull in the external ftl-injector role.
+# This might be enabled when we have solvers to run or graders for ILT
+install_ftl: false
+
+# TODO: Decide on whether to use sat or give access to repos directly with key
+# This will tell Agnosticd to use either:
+# satellite, rhn, or file for repos
+# Only satellite is supported for RHEL 8
+repo_method: satellite
+
+# If using satellite, these are needed and should
+# come from secrets:
+# satellite_url: satellite.opentlc.com
+# satellite_org: # This should be stored in secrets
+# satellite_activationkey: # This should be stored in secrets
+
+# use_content_view: true
+# If using file, these are needed in addition to the repos_template.j2 file:
+osrelease: '4.4.0'
+repo_version: '4.4'
+
+# own_repo_path: points to a repo mirror. Must defined in secrets
+#own_repo_path: <FROM_SECRETS>
+
+# Packages to install on all of the hosts deployed as part of the agnosticd config
+# This invokes the "common" role
+install_common: true
+
+# As part of the "common" role, this cause it to do a yum update on the host
+update_packages: true
+
+# Number of Nodes
+master_instance_count: 3
+worker_instance_count: 3
+
+# Install OpenShift 4 - and which version
+install_ocp4: true
+ocp4_installer_version: "4.4.9"
+
+# Run logic to enable cluster shutdown before 24h initial certificate rotation 
+# Only works for OCP 4.1 and 4.2. OCP 4.4 should no longer require this.
+ocp4_enable_cluster_shutdown: false
+
+# Install an OpenShift 4 Dev Preview Release
```

Comment:
If you're not using any of these vars, please clean them up rather than leaving them commented.

Reply:
done